sqlite/.gitignore

```diff
@@ -0,0 +1,4 @@
+workdir/
+firecracker.socket
+initrd.cpio
+.config
```

Comment:
Add ending newline (but NOT a new blank line, just a newline character at the end).